src/test/kotlin/org/treeWare/mySql/operator/CreateDatabaseTests.kt

```diff
@@ -17,19 +16,23 @@ import kotlin.test.assertEquals
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class CreateDatabaseTests {
-    private val dbServer: MySQLContainer<Nothing>
     private val connection: Connection
 
     init {
-        dbServer = MySQLContainer<Nothing>(DockerImageName.parse("mysql:8.0.29"))
-        dbServer.start()
-        connection = DriverManager.getConnection(dbServer.jdbcUrl, "root", dbServer.password)
+        connection = DriverManager.getConnection(
```

Comment:
nit: create a getConnection() method in the singleton to return a connection to the database. Might also be worth passing in an autoCommit boolean argument to the new method.